frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -348,10 +370,99 @@ export function OutputSegmentationLayerNameFormItem({
   );
 }
 
-function StartingJobModal(props: StartingJobModalProps) {
+export function StartAIJobModal({ aIJobModalState }: StartAIJobModalProps) {
+  const centerImageStyle = {
+    margin: "auto",
+    width: 150,
+  };
+  return aIJobModalState !== "invisible" ? (
+    <Modal
+      width={667}
+      open
+      title={
+        <>
+          <i className="fas fa-magic" />
+          Automated analysis with WEBKNOSSOS
```

Comment:
I asked in Slack whether this title should be adapted. In any case, I'd use all-caps.

Reply:
(note-to-self: simply use "AI analysis" as suggested)